src/App.js

```diff
@@ -1,17 +1,48 @@
 import React from 'react';
 import './App.css';
-import Header from '../src/components/header/header';
-import Form from '../src/components/form/form';
-import Footer from '../src/components/footer/footer';
-
-function App() {
-  return (
-    <div className="App">
-      <Header />
-      <Form />
-      <Footer />
-    </div>
-  );
+import Header from './components/header/Header';
+import Form from './components/form/Form';
+import Results from './components/results/Results';
+import Footer from './components/footer/Footer';
```

Comment:
For all these files you are going to need to have the .js file name lower case because it is looking for a file that is capitalized and then can't find it